### ISSUE TITLE
Run crichere 

### ISSUE THREAD

**Issue by darshansharma** [darshansharma/CricHere#3]
Run Crichere on your system and paste a screenshot of whether it runs or not?

**Comment by ramchandarkg** [comment]
Hi @darshansharma could you pls assign me in this. I'm interested to work in this for #hacktoberfest.

**Comment by ramchandarkg** [comment]
I ran crichere on my system and this is the error i am getting.

![screenshot](https://github.com/darshansharma/CricHere/assets/55615436/835a048e-b280-44b7-b359-8e98606014fb)

**Comment by darshansharma** [comment]
@ramchandarkg Yeah, It might be because cricapi no longer exists. Also parsing of string might be failing due to that
Need to do proper error handling.

**Comment by ramchandarkg** [comment]
Could you assign me in the other issue regarding finding new api and integrate it. I'm looking forward to contribute to it.

**Comment by darshansharma** [comment]
Head over to [Issue 5](https://github.com/darshansharma/CricHere/issues/5)
